examples/ui_actions_explorer/public/plugin.tsx

```diff
@@ -46,21 +54,43 @@ export class UiActionsExplorerPlugin implements Plugin<void, void, {}, StartDeps
 
     const startServices = core.getStartServices();
 
-    deps.uiActions.addTriggerAction(
-      USER_TRIGGER,
+    deps.uiActions.addTriggerActionAsync(USER_TRIGGER, TRIGGER_PHONE_TRIGGER_ACTION, async () =>
       createTriggerPhoneTriggerAction(async () => (await startServices)[1].uiActions)
     );
-    deps.uiActions.addTriggerAction(
-      USER_TRIGGER,
+    deps.uiActions.addTriggerActionAsync(USER_TRIGGER, EDIT_USER_ACTION, async () =>
       createEditUserAction(async () => (await startServices)[0])
     );
 
-    deps.uiActions.addTriggerAction(COUNTRY_TRIGGER, viewInMapsAction);
-    deps.uiActions.addTriggerAction(COUNTRY_TRIGGER, lookUpWeatherAction);
-    deps.uiActions.addTriggerAction(COUNTRY_TRIGGER, showcasePluggability);
-    deps.uiActions.addTriggerAction(PHONE_TRIGGER, makePhoneCallAction);
-    deps.uiActions.addTriggerAction(PHONE_TRIGGER, showcasePluggability);
-    deps.uiActions.addTriggerAction(USER_TRIGGER, showcasePluggability);
+    deps.uiActions.addTriggerActionAsync(
+      COUNTRY_TRIGGER,
+      VIEW_IN_MAPS_ACTION,
+      async () => viewInMapsAction
+    );
+    deps.uiActions.addTriggerActionAsync(
+      COUNTRY_TRIGGER,
+      LOOK_UP_WEATHER_ACTION,
+      async () => lookUpWeatherAction
```

Comment:
Non of these seem to be really async. imo we should either revert or make this async and lazy load properly to avoid bad examples